examples/plotting_with_geoplot.py

```diff
@@ -14,75 +14,77 @@
 
 First we'll load in the data using GeoPandas.
 """
-import geopandas
-
-path = geopandas.datasets.get_path('naturalearth_lowres')
-df = geopandas.read_file(path)
-# Add a column we'll use later
-df['gdp_pp'] = df['gdp_md_est'] / df['pop_est']
-
-boroughs = geopandas.read_file(geopandas.datasets.get_path('nybb')).to_crs(epsg='4326')
-injurious_collisions = geopandas.read_file(
-    "https://github.com/ResidentMario/geoplot-data/raw/master/nyc-injurious-collisions.geojson")
+import geopandas as gpd
+import geoplot as gplt
```

Comment:
Not sure whether it is not better to use the same principle as above. Especially when you do `import geoplot.crs as gcrs` below, it gets unnecessarily more complicated to understand.

Reply:
`cartopy` has its own `crs` submodule, `cartopy.crs`, which by convention is imported `ccrs`. `geoplot` copies this convention, importing `geoplot.crs` as `gcrs`. What do you suggest as an alternative? `import geoplot.crs as gcrs` is already in the current version of these docs.

Reply:
I know it is in the current version. I would use `projection=geoplot.crs.Orthographic()` as it is self-explanatory and user do not have to look in the document what `gcrs` is. But if you prefer to keep `gcrs` as a convention, I do not mind much.

Reply:
Ah, I see what you mean. I've updated it to `geoplot.crs.Orthographic()` (and done likewise with the other projection parameters).